packages/next/src/server/web/types.ts

```diff
@@ -27,7 +27,7 @@ export interface RequestData {
   }
   page?: {
     name?: string
-    params?: { [key: string]: string | string[] }
+    params?: { [key: string]: string | string[] | undefined }
```

Comment:
This one jumps out. Why is `undefined` a possible value here?

Reply:
It makes it compatible with the `Params` type which is `Record<string, string | string[] | undefined>` (as we don't warn on unchecked index access)